packages/babel-preset-env/CONTRIBUTING.md

```diff
@@ -38,19 +38,31 @@ and split it with `/`:
 
 `Object static methods / Object.values`
 
-Find the corresponding module on [core-js](https://github.com/zloirock/core-js/tree/master/modules):
+Find the corresponding module on [`core-js@2`](https://github.com/zloirock/core-js/tree/v2/modules):
 
-`es7.object.values.js`
+`es.object.values.js`
 
-Find required ES version in [`built-in-features.js`](https://github.com/babel/babel/blob/master/packages/babel-preset-env/data/built-in-features.js) and add the new feature:
+Find required ES version in [`corejs2-built-in-features.js`](https://github.com/babel/babel/blob/master/packages/babel-preset-env/data/corejs2-built-in-features.js) and add the new feature:
 
 ```js
-const es2017 = {
+const es = {
   //...
-  "es7.object.values": "Object static methods / Object.values"
+  "es6.object.values": "Object static methods / Object.values"
```

Comment:
This is es7: https://github.com/zloirock/core-js/blob/e778e8026aed8a58f93f1ee4e3192cd1a7d7bdf5/modules/es7.object.values.js

Reply:
Yep, seems I made some typos when I revert it to `core-js@2`.